index.js

```diff
@@ -2,7 +2,16 @@ const http = require('http')
 const rangeParser = require('range-parser')
 const mime = require('mime-types')
 
-module.exports = async function serve (drive, opts = {}) {
+module.exports = async function serve (drives, opts = {}) {
+  let defaultDrive = null
+
+  if (drives instanceof Map) {
+    defaultDrive = drives.entries().next().value[1]
```

Comment:
I think this will error out when passed an empty map (which I suppose is okay if that's not allowed. Makes sense to expect at least one drive before launching the server).

Reply:
Yep, it could make sense to accept an empty Map so I can improve that part

Just realized I should check if drive wasn't found after the `drives.get(id)` or server will crash